_notices/rsn0041.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 41 # should match notice number
+notice_pin: true # set to true to pin to notice page
+
+title: "Remove cugraph-ops from RAPIDS"

```

Comment:
```suggestion
title: "Removing cugraph-ops from RAPIDS"
```

I think "removing" is a bit more consistent with the way the other things at https://docs.rapids.ai/notices/ are written.